src/controllers/userControllers.js

```diff
@@ -14,17 +16,26 @@ class UserManager {
    */
   static async registerUser(req, res) {
     try {
-      const { username, email, password } = req.body;
+      const {
+        username, email, password, bio, image
```

Comment:
Why adding bio and image when signing up?
I think you should remove bio and image